doc/source/setup-helm.rst

```diff
@@ -38,7 +38,11 @@ cluster. At the terminal, enter:
 
 2. Give the ``ServiceAccount`` `RBAC
    <https://kubernetes.io/docs/admin/authorization/rbac/>`_ full permissions to
-   manage the cluser. While most clusters have RBAC enabled and you need this
+   manage the cluser.
+
+   **If you're on Azure AKS, you should skip directly to step 3.**
```

Comment:
Shouldn't this be at the end of step 1 and not step 2? I'm happy either way.

Reply:
good point - updated